peerconnection.go

```diff
@@ -356,7 +360,7 @@ func (pc *PeerConnection) checkNegotiationNeeded() bool {
 			// Step 5.3.1
 			if t.Direction() == RTPTransceiverDirectionSendrecv || t.Direction() == RTPTransceiverDirectionSendonly {
 				descMsid, okMsid := m.Attribute(sdp.AttrKeyMsid)
-				if !okMsid || descMsid != t.Sender().Track().Msid() {
+				if !okMsid || t.Sender() == nil || descMsid != t.Sender().Track().Msid() {
```

Comment:
this is not enough. `t.Sender()` can become nil between the nil check and accessing `Track().Msid()`